### PR TITLE
Restaaurant page using webpack

### PR DESCRIPTION
**Pull request by widzthedvloper**
In this pull request I complete the following works:
-Creation of HTML elements using DOM
-Webpack configuration
-Linter set up
-Initialize npm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "myrestaurant",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "build": "webpack"
+  },
```
Comment (afanimartin) [inline package.json:10]: - Add the necessary commands to the scripts section
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import insertNavbar from './homePage';
+import homeContent from './homeContent';
+import insertMenu from './menu';
+import insertForm from './contact';
+import './style.css';
+
+insertNavbar();
+homeContent();
+
+function removeContent() {
```
Comment (afanimartin) [inline src/index.js:10]: - Use ES6 arrow syntax.
```javascript
const removeContent = () => {}
```

File: src/menu.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+const container = document.getElementById('content');
+
+// eslint-disable-next-line require-jsdoc
+function insertMenu() {
```
Comment (afanimartin) [inline src/menu.js:4]: - Use ES6 syntax

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required :pray:
Nice work on this PR. Your app looks good and works and works well.
Features are well implemented using modules.
Let's however, make some changes so we can merge this PR.
- [ ] Use ES^ syntax throughout the codebase. You currently have the `function` keyword.
- [ ] Add the necessary commands to the `scripts` section of the `package.json` file. Check my comments for suggestions.


### Martin Afani
#### Slack: whiz25

**Review by oloomoses — APPROVED** [review]
# Congratulations, Project Approved. :muscle: :tada: :clap: 
**Hello @widzthedvloper,
Great job working on the suggested changes from the previous review.
You have passed all the requirements for this project :muscle: 
Feel free to merge and close this pull request.**

**I wish you the best in your next project and see you at the top :rocket:** 
#### Happy Coding
_Slack: Oloo Moses_

**Pull request merged**